src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -189,90 +129,59 @@ serve stylesheets and fonts from separate origins. For example, this is how the
 
-Generally speaking, using the
-[`preload`](https://developer.mozilla.org/docs/Web/HTML/Preloading_content)
-resource hint to load fonts should be avoided. Although `preload` is highly
-effective at making fonts discoverable early in the page load process, this
-comes at the cost of taking away browser resources from the loading of other
-resources.
-
-In most scenarios, inlining font declarations and adjusting stylesheets is
-a more effective approach. These adjustments come closer to addressing the root
-cause of late-discovered fonts - rather than just providing a workaround.
-
-In addition, using `preload` as a font-loading strategy should also be used
-carefully as it bypasses some of the browser's built-in content negotiation
-strategies. For example, `preload` ignores `unicode-range` declarations, and if
-used prudently, should only be used to load a single font format.
+Inlining font declarations and adjusting stylesheets may be a more effective approach. These adjustments come closer to addressing the root cause of late-discovered fonts—rather than just providing a workaround.
 
+In addition, using `preload` as a font-loading strategy should also be used carefully as it bypasses some of the browser's built-in content negotiation strategies. For example, `preload` ignores `unicode-range` declarations, and if used prudently, should only be used to load a single font format.
 
+However, when using external stylesheets, preloading the most important fonts can be very effective since the browser will not otherwise discover whether the font is needed until much later.
 
 ## Font delivery
 
-Faster font delivery yields faster text rendering. In addition, if a font is
-delivered early enough, this can help eliminate layout shifts resulting from
-font swapping.
-
+Faster font delivery yields faster text rendering. In addition, if a font is delivered early enough, this can help eliminate layout shifts resulting from font swapping.
 
+### Using self-hosted fonts
 
-### Best practices
+On paper, using a self-hosted font should deliver better performance as it eliminates a third-party connection setup. However, in practice, the performance differences between these two options is less clear cut: for example, the [Web Almanac](https://almanac.httparchive.org/en/2020/fonts#fig-7) found that sites using third-party fonts had a faster render than fonts that used first-party fonts.
 
+If you are considering using self-hosted fonts, confirm that your site is using a [Content Delivery Network (CDN)](/content-delivery-networks/) and [HTTP/2](/content-delivery-networks/#http2-and-http3). Without use of these technologies, it is much less likely that self-hosted fonts will deliver better performance. For more information, see [Content Delivery Networks](/content-delivery-networks/).
 
-#### Using self-hosted fonts
+{% Aside %}
```

Comment:
```suggestion
{% Aside 'caution' %}
```

Reply:
I think `'caution'` is too much here.

<img width="799" alt="image" src="https://user-images.githubusercontent.com/10931297/194283679-39b3e612-d77a-4fe6-afd3-a29df879ca50.png">

Considered `'important'`, but even then it's debatable which is better. Self-hosting a slow font can still be better than third-partying a faster font.

Will leave as is.